enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -3,9 +3,10 @@ import styled from "@emotion/styled";
 import { baseStyle, getRootStyle } from "metabase/css/core/base.styled";
 import { alpha, color } from "metabase/lib/colors";
 import { aceEditorStyles } from "metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.styled";
+import { defaultFontFiles } from "metabase/styled-components/fonts";
```

Comment:
So I'm currently putting these kind of styled component (emotion) files closer to the CSS files itself.

https://github.com/metabase/metabase/blob/5d436bda63a233a3365bf0635e69e1a3a83408e9/frontend/src/metabase/css/core/base.styled.ts#L4

It seems that this approach looks similar to what I did there. I feel like they should live closer together. I don't have a strong conviction where they should live. But maybe you'll have some ideas?

Reply:
I'm don't really have a strong opinion on this one either, but I'm up for putting these files next to each other, maybe in a `fonts.styled.ts` next to `base.styled.ts`

Reply:
ok, moved. it makes more sense than where it is right now